app/src/main/res/values/strings.xml

```diff
@@ -1601,6 +1601,10 @@
     <string name="login_deletion_confirmation">Are you sure you want to delete this login?</string>
     <!-- Positive action of a dialog asking to delete  -->
     <string name="dialog_delete_positive">Delete</string>
+    <!-- Text for the OK button on dialog -->
+    <string name="dialog_button_ok">OK</string>
+    <!-- Text for the cancel button on dialog -->
+    <string name="dialog_button_cancel">Cancel</string>
```

Comment:
@Delphine Can you check if this is ok? I know in the past I tried to also make a generic OK and Cancel string, but I think the recommendation then and now is still we want an unique string for each of the components so there is accurate description of how and where the string is used.

<img width="1051" alt="Screen Shot 2022-03-02 at 9 33 28 AM" src="https://user-images.githubusercontent.com/1190888/156382166-efd2b9a6-5d98-42ca-9bfc-e35ebb17309a.png">

Reply:
Correct, the recommendation is to have a unique string for each of the components. thanks for asking!